data/README.txt

```diff
@@ -0,0 +1,3 @@
+Use this command from DogShow/ to import the data from puppy_seed.json
+
+mongoimport --db dogshow --collection dogs --drop --file data/puppy_seed.json --type json --jsonArray

```

Comment:
depends on if you want it to reset all the scores and wins/loss counters everytime you do npm start